paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ComposeFormDataCollectionFragment.kt

```diff
@@ -46,6 +46,18 @@ class ComposeFormDataCollectionFragment : Fragment() {
             ViewGroup.LayoutParams.MATCH_PARENT
         )
 
+        formViewModel.setSaveForFutureUse(
+            requireArguments().getBoolean(
+                EXTRA_SAVE_FOR_FUTURE_USE_VALUE
+            )
+        )
+
+        formViewModel.setSaveForFutureUseVisibility(
+            requireArguments().getBoolean(
+                EXTRA_SAVE_FOR_FUTURE_USE_VISIBILITY
+            )
+        )
+
```

Comment:
This should be in `onCreate`, or even passed in the constructor since it's just the initial value. Here it'll be set again if the view is recreated, losing the state of `SaveForFutureUse`.

Reply:
The constuctor seems better